include/interaction/Simulation.h

```diff
@@ -0,0 +1,95 @@
+#ifndef ANIMATOR_H
+#define ANIMATOR_H
+
+#include "choreograph/Choreograph.h"
+
+#include <glm/glm.hpp>
+
+#include "PointOfInterests.h"
+
+
+
+enum ANIMATION_STATE
+{
+  STOP,
+  PLAYING,
+  PAUSE
+};
+
+struct SimulationState
+{
```

Comment:
is the simulation the same as the animation?

Reply:
No.  Animation is the transition between simulation states.

Reply:
could you document that in the readme and/or comments?